docs/requirements.txt

```diff
@@ -22,7 +22,7 @@ ghp-import==2.0.1
     # via mkdocs
 gitdb==4.0.7
     # via gitpython
-gitpython==3.1.24
+gitpython==3.1.18
```

Comment:
I don't understand how did we had the `3.1.24` version if the latest one is `3.1.20`, maybe they removed versions from pypi? I thought that was not possible.

If the latest version is `3.1.20` why did you choose `3.1.18`?

Reply:
Latest is 3.1.24 (see [here](https://pypi.org/project/GitPython/3.1.24/)). The problem is that they removed support for python 3.6 in 3.1.23 and all tags from 3.1.19 until 3.1.24 are yanked releases. That's why I decided to go with the latest proper release before this breaking change, which was 3.1.18.
Anyway, I was still testing it :sweat_smile: I intend to send you the PR until build was passing, which I couldn't do yesterday because coveralls is down (and [still is](https://coveralls.io/) :sob: ), but I forgot to mark it as draft. Sorry about that :see_no_evil:

Reply:
Python 3.6 is quite old, I feel we can drop support for it too if that helps us.

Reply:
Yeah, I think that would be the best solution, since now it's just gitpython but soon enough other dependencies will start removing support for py36 too.

Reply:
We could also add support for 3.9 :smirk: